components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -495,6 +497,14 @@ class NewTabPage extends React.Component<Props, State> {
       return null
     }
 
+    const customMenuItems = [
+      {
+        label: 'rewardsOpenPanel',
+        renderIcon: () => { return <BatIcon/> },
```

Comment:
Just curious - do you think we can/should use the "leo" icon `product-bat-color`? It's a bit more work since you have to set the icon size with a CSS variable instead of directly in the svg React code, but might be preferred to adding a new icon file.

Reply:
That makes sense, I'll add that (though probably `product-bat-outline` for this specific case).

Reply:
Fixed now.